.pipelines/e2e.yml

```diff
@@ -1,3 +1,19 @@
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
+
+pr:
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
```

Comment:
Explicit run when pr to `master` created with omit of `docs/*` folder.
Implicit is all branches all files.

Reply:
could you add README.md (root of this repo) to this as well? Or make this all markdown files?